gooddata-java-model/src/main/java/com/gooddata/sdk/model/settings/SettingItem.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2007-2021, GoodData(R) Corporation. All rights reserved.
```

Comment:
Please use the same copyright header as there is in the rest of source files in this repo (FYI: you can have multiple profiles in idea - one for private repos and another one for open source ;) )

Reply:
Changed.